src/engine/enginevumeter.cpp

```diff
@@ -93,6 +99,24 @@ void EngineVuMeter::process(CSAMPLE* pIn, const int iBufferSize) {
     } else {
         --m_peakDuration;
     }
+
+    if (clipped & 1) {
```

Comment:
> The clipped bitmap is the input. You calculate 3 cos from it while one is should be the result of the output of the L OR R :m_ctrlPeakIndicatorR || m_ctrlPeakIndicatorL.

The way it is now, it is in fact equivalent to m_ctrlPeakIndicatorR || m_ctrlPeakIndicatorL.

It is true that it won't hurt to redundantly calculate clipped as m_ctrlPeakIndicatorR || m_ctrlPeakIndicatorL for the sake of clarity. But a comment would clarify things as well.

If you prefer the way you proposed rather than a comment, I'll address it. :)